src/helpers/utils.ts

```diff
@@ -80,6 +80,12 @@ export function getStrategy(strategy, spaces) {
   return strategy;
 }
 
+export function switchStrategyAt(space, proposal) {
+  const timeNow = Date.now();
```

Comment:
With the code now it will always return de proposal strategies, it should be based on the proposal timestamp not on the current timestamp  :)